kod/object/active/holder/nomoveon/battler/player.kod

```diff
@@ -12226,6 +12235,30 @@ messages:
       return;
    }
 
+   ExpireQuestByTemplate(questTemplate = $)
```

Comment:
I'm thinking I'll refactor this into QuestEngine as `ExpireQuest(index=0, who=$)`